.changeset/wild-trees-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+"comet-site-v9": major
```

Comment:
why is this a major change? shouldnt this be a minor?

Reply:
Version was changed.

e82cf5ca12118572e1d54e3ada7ae19db01fc69f

Reply:
Changed charset doc

ddcba888110cf58e927ea0fda07ef981a430dd64